core/src/main/java/org/mule/runtime/core/construct/AbstractPipeline.java

```diff
@@ -416,6 +428,10 @@ protected void doDispose() {
     super.doDispose();
   }
 
+  protected boolean useBlockingCodePath() {
```

Comment:
`isBlocking` for sure if this was a field being exposed via a javabeans getter/setter.  But it is an internal behavioural method and `use` is just as good as `should` or `shouldUse` for this.